Description: Correct POD issues in LedgerSMB/ScriptLib/Company.pm
 Spelling error in line 213: informations information
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472498&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/ScriptLib/Company.pm
+++ b/LedgerSMB/ScriptLib/Company.pm
@@ -210,7 +210,7 @@
 
 Extracts a single company from the database, using its company ID as the primary
 point of uniqueness. Shows (appropriate to user privileges) and allows editing
-of the company informations.
+of the company information.
 
 =back
 
